package.json

```diff
@@ -39,10 +39,10 @@
     "@a11y/focus-trap": "^1.0.5",
     "bulma": "^0.8.0",
     "csz": "^1.2.0",
-    "date-fns": "^2.11.0",
+    "date-fns": "^2.15.0",
     "htm": "^3.0.3",
     "lodash-es": "^4.17.15",
-    "preact": "^10.3.2",
+    "preact": "^10.4.6",
```

Comment:
Is it safe to update our libs? Do we have tests to make sure that nothing had conflicts?

Reply:
I test it, and works fine, but we will need add some unit test to be sure